pytorch_lightning/trainer/training_loop.py

```diff
@@ -457,7 +457,7 @@ def _track_gradient_norm(self):
         if (self.global_step + 1) % self.trainer.log_every_n_steps == 0:
             if float(self.trainer.track_grad_norm) > 0:
                 model = self.trainer.lightning_module
-                grad_norm_dict = grad_norm(model, self.trainer.track_grad_norm)
+                grad_norm_dict = grad_norm(model, self.trainer.track_grad_norm, self.trainer.folder_separator)
```

Comment:
There is the access to the logger missing.

Reply:
Fixed, sorry I was in a hurry and I didn't proof check